indexer/editable_map_object.cpp

```diff
@@ -496,27 +497,27 @@ void EditableMapObject::SetWebsite(string website)
 
 void EditableMapObject::SetFacebookPage(string facebookPage)
 {
-  m_metadata.Set(feature::Metadata::FMD_CONTACT_FACEBOOK, facebookPage);
+  m_metadata.Set(feature::Metadata::FMD_CONTACT_FACEBOOK, ValidateAndFormat_facebook(facebookPage));
```

Comment:
All these methods take `string const &`. Then you don't need to create a copy of the string in the function argument by using `string facebookPage` (yes, in C++ this declaration creates a copy of the passed parameter). Use `string const & facebookPage` here and in other functions.

Reply:
Changed argument type